src/server/run-websocket-server.ts

```diff
@@ -1,12 +1,27 @@
+import * as express from "express";
 import * as WebSocket from "ws";
 import * as Knex from "knex";
+import * as http from "http";
 import { JsonRpcRequest } from "../types";
 import { isJsonRpcRequest } from "./is-json-rpc-request";
 import { dispatchJsonRpcRequest } from "./dispatch-json-rpc-request";
 import { makeJsonRpcResponse } from "./make-json-rpc-response";
+import { ErrorCallback } from "./../types"
 
-export function runWebsocketServer(db: Knex, port: number): void {
-  const websocketServer: WebSocket.Server = new WebSocket.Server({ port });
+let app: any = express();
+let server: http.Server  = http.createServer(app);
+
+
+app.get('/', (req: any, res: any): void => {
+  res.send("Hello");
+});
+
+export function runHttpServer(port: number, callback: ErrorCallback): void {
```

Comment:
Why are we setting up an HTTP server?  MVP augur-node is intended to be websockets-only.  Is there something we need an HTTP server for?

Reply:
Went ahead and removed the HTTP server stuff so this can be merged -- if heroku needs this for some reason we can make it a separate PR.